### DISCUSSION TITLE
현재 작업 방식 검토

### DISCUSSION THREAD

**Discussion by fkdl0048** [The-Day-In-1984/1984_Project#47] [General]
1. 현재 작업 방식 검토

풀리퀘 사용법이나 이슈 사용법에 대한 토론

현재 방식을 그대로 가져가기로 함.

Discssion에 올려둔 내용을 참고..!

2. 자신의 작업 주 단위 기록

현재 작업 추적은 문제없이 진행되고 있으나

각자 정한 마감일이 언제인지 몰라서 서로 확인이 불가능

각자 정한 작업에 대한 예상되는 마감일을 기록 

* 작업 시작 전 주에 다음주가 바빠서 작업이 불가능하다면 작업 내용이 없어야 함
* 스스로 판단해서 작업을 내용을 적는게 중요..!
* 하루 2시간씩 일주일 작업을 한다고 하고 못지키는 것 보다 전 주에 다음주에 대한 작업시간이 0시간인 것이 훨씬 좋은 방법

따라서 각자 작업에 대한 목표일정도를 적어주세요

**Comment by fkdl0048** [comment]
@ccmuk58 @kyoonge

**Comment by ccmuk58** [comment]
각 작업에 대한 목표일은 작업 마다 바인딩 된 이슈에 기록하면 될까요?
아니면 이곳에?!

**Reply by fkdl0048** [reply]
작업이슈에 시작일과 종료일 정도로 기록해두면 나중에 추적하기 좋을 것 같아요..!

**Reply by fkdl0048** [reply]
![image](https://user-images.githubusercontent.com/84510455/228109520-b1bba63d-291a-4da3-b001-b952c61a4e4b.png)

정리되니까 훨씬 보기 좋네요..!
